src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -392,12 +401,6 @@ void EIRPlantLoopHeatPump::getDesignCapacities(
     EnergyPlusData &state, const PlantLocation &calledFromLocation, Real64 &MaxLoad, Real64 &MinLoad, Real64 &OptLoad)
 {
     if (calledFromLocation.loopNum == this->loadSidePlantLoc.loopNum) {
-        this->sizeLoadSide(state);
-        if (this->waterSource) {
-            this->sizeSrcSideWSHP(state);
-        } else if (this->airSource) {
-            this->sizeSrcSideASHP(state);
-        }
```

Comment:
The `referenceCapacity` needed by `sizeSrcSideASHP` to size `sourceSideDesignVolFlowRate` is only set when `state.dataPlnt->PlantFirstSizesOkayToFinalize` is `true` which was never the case when `EIRPlantLoopHeatPump::getDesignCapacities` was called which resulted in the `referenceCapacity` used for sizing calculations to always be -99999, which always resulted in a negative autosized source side air flow rate. The proposed changes move this block in `EIRPlantLoopHeatPump::onInitLoopEquip` which gets called through the different passes of `InitializeLoops` when `state.dataPlnt->PlantFirstSizesOkayToFinalize` is set to true.

`ChillerElectricEIR` follows a similar approach.